src/main/java/com/game/Battleship/models/Board.java

```diff
@@ -0,0 +1,40 @@
+package com.game.Battleship.models;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Board {
+
+    private int defaultBoardSize = 10;
+    private int size;
+    private List<Cell> boardField;
+
+    public Board(){
+        this.size = defaultBoardSize;
+        this.boardField = generateBoardField();
+
+    }
+
+    public List<Cell> getBoardField() {
+        return boardField;
+    }
+
+    public void setBoardField(List<Cell> boardField) {
+        this.boardField = boardField;
+    }
+
+    public int getSize(){
+        return size;
+    }
+
+    public void setSize(int size){
+        this.size = size;
+    }
+
+    private List<Cell> generateBoardField(){
+
+        List<Cell> result = IntStream.range(0, (int) Math.pow(this.size, 2)).mapToObj(i -> new Cell()).collect(Collectors.toList());
```

Comment:
Wow this one line is doing a lot. I would like to see this broken up into pieces that explains what each piece is doing.  That will make it easier for someone who comes behind you to know what this bit of code is doing. I see a couple of pieces you could work with:

```
.range(0, (int) Math.pow(this.size, 2))
.mapToObj(i -> new Cell())
.collect(Collectors.toList())
```

This would make your code more expressive to what it is actually doing.

Reply:
Done